### PR TITLE
Yaswanth/ refactor: lazy loading cfd

### PR DESCRIPTION
**Pull request by yaswanth-deriv**
Changes:

- Refactor: lazy loading cfd

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/trade-types/cfds/index.tsx [hunk 1]
```diff
@@ -1,18 +1,81 @@
...
+import { useHandleLazyLoad } from 'components/hooks/use-handle-lazy-load'
+const WhatIsCFD = Loadable(() => import('./_what_is_cfd'))
 const WhyTradeCFD = Loadable(() => import('./_why-trade-cfd'))
 const TradingCFDIncreases = Loadable(() => import('./_trading-cfd-increases'))
 const StartTrading = Loadable(() => import('./_start-trading'))
 const ThingsToKeep = Loadable(() => import('./_mind-when-trading'))
 const AvailableMarkets = Loadable(() => import('./_available-markets'))
 
 const CFD = () => {
+    const target = '#gatsby-focus-wrapper'
```
Comment (niloofar-deriv) [inline src/pages/trade-types/cfds/index.tsx:25]: It's better to move `target`, `options` and `lazy_components` and `lazyTemplate` outside the CFD!

File: src/pages/trade-types/cfds/index.tsx [hunk 2]
```diff
@@ -1,18 +1,81 @@
...
+    const options = {
+        root: null,
+        rootMargin: '0px',
+        threshold: 0.7,
+    }
+    const lazy_components: ReactNode = (
```
Comment (niloofar-deriv) [inline src/pages/trade-types/cfds/index.tsx:31]: Just for clarification, Do we need to define type for this?

**Review by niloofar-deriv — COMMENTED** [review]
Just a reminder, did u minify the images with `svgo`?

File: src/pages/trade-types/cfds/index.tsx [hunk 3]
```diff
@@ -1,18 +1,81 @@
...
+        <>
+            <WhatIsCFD
+                fallback={
+                    <div>
+                        <img src={CFD_DUMMY} style={{ width: '95%' }} alt="trade-type" />
```
Comment (ashraf-deriv) [inline src/pages/trade-types/cfds/index.tsx:36]: Please update the "alt" values appropriately for all the fall back images down below.

File: src/pages/trade-types/cfds/index.tsx (hunk 1 above)
Comment (yaswanth-deriv) [reply src/pages/trade-types/cfds/index.tsx:25]: Yeah  sure i will keep it outside the CFD

**Comment by ashraf-deriv** [comment]
> Just a reminder, did u minify the images with `svgo`?
@niloofar-deriv  This images are .png right?

**Pull request closed without merging**